imagetagger/imagetagger/annotations/templates/annotations/annotate.html

```diff
@@ -252,7 +252,6 @@ <h3 id="active_image_name" class="panel-title">{{selected_image.name}}</h3>
                               </div>
                             {% endif %}
                             {% if 'delete_images' in imageset_perms and not imageset_lock %}
-                            <div class="col-sm-12"><p>{{ imageset.image_lock }}</p></div>
```

Comment:
Removing the variable makes sense (I don't know why I ever put it there). But please keep the surrounding `div` and `p` for the layout.

Reply:
Updated.